### PR TITLE
Add mapfiles.d dirs to HTCondor-CE mapfile (HTCONDOR-244)

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: config/condor_mapfile [hunk 1]
```diff
@@ -1,31 +1,18 @@
...
-# callout mechanism (which may involve plugins such as LCMAPS or Argus).
-GSI (.*) GSS_ASSIST_GRIDMAP
-SSL "/CN=([-.A-Za-z0-9/= ]+)" \1@unmapped.htcondor.org
-CLAIMTOBE .* anonymous@claimtobe
-FS "^(root|condor)$" \1@daemon.htcondor.org
-FS "(.*)" \1
+###############################################################################
+
+# Place local authentication mapping customizations
+# in /etc/condor-ce/config.d
```
Comment (johnkn) [inline config/condor_mapfile:14]: I think this comment is wrong?

**Comment by brianhlin** [comment]
@matyasselmeci can you take a look over the packaging changes?

File: config/condor_mapfile (hunk 1 above)
Comment (brianhlin) [reply config/condor_mapfile]: Yup, fixed

**Review by matyasselmeci — APPROVED** [review]

**Comment by brianhlin** [comment]
@matyasselmeci @johnkn requesting re-review since I added a commit to set `CERTIFICATE_MAPFILE_ASSUME_HASH_KEYS = True`, which changes the format of mapfiles so that regex in the second field have to be enclosed by `/`

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**